homeassistant/components/fluss/button.py

```diff
@@ -0,0 +1,53 @@
+"""Support for Fluss Devices."""
+
+import logging
+
+from fluss_api.main import FlussApiClient
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+LOGGER = logging.getLogger(__package__)
+DEFAULT_NAME = "Fluss +"
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up the Fluss Devices."""
+
+    api: FlussApiClient = entry.runtime_data
```

Comment:
will be typed impolicitly when the you type the configentry
```suggestion
    api = entry.runtime_data
```